src/engine/filters/enginefilterdelay.h

```diff
@@ -29,6 +29,13 @@ class EngineFilterDelay : public EngineObjectConstIn {
 
     void setDelay(unsigned int delaySamples) {
         m_delaySamples = delaySamples;
+
+        // When mixxx will support other channel count variants than stereo,
```

Comment:
You may put a static_assert for this condition as well. That will fail the build in that case.

Reply:
Can you also assert here that the delay is devide-able by the channel count? 
This way we can below just use `< SIZE` which is easier to understand.

Reply:
Agree, that is really more clear way.